### ISSUE TITLE
Dynamic firewall changes?

### ISSUE THREAD

**Issue by MuschPusch** [omega8cc/boa#575]
We have gitlab running on a different port (2222) and it's blocked by the current firewall settings.  Wouldn't be nice to have some control file for dynamic firewall changes?

(Temporary workaround for people having similar issues create a .netrc and use http)

**Comment by omega8cc** [comment]
You can modify your csf settings for open ports but it can't and shouldn't be done "dynamically".

**Comment by MuschPusch** [comment]
So no fix possible when using omega8cc hosting?

**Comment by omega8cc** [comment]
It is not possible technically on our hosted service because we don't manage firewall on the instance level, rather on the network level, so we can't allow to introduce any overrides or customizations.

**Comment by MuschPusch** [comment]
ok sounds reasonable. Thanks for the quick response
